code/src/nuvla/connector/kubernetes.py

```diff
@@ -169,15 +169,84 @@ def list(self, filters=None):
         # return execute_cmd(cmd)
         pass
 
+    def _get_container_logs(self, namespace, values, since_opt, lines: int) -> str:
+        tail_lines=lines # wtf
+        tail_lines=5 # wtf
+        logs_string=str()
+        pod_unique_id = str(values["metadata"]["name"])
+        log.debug('Unique pod ID: %s ', pod_unique_id)
+        try:
+            for containers_list in values['spec']['containers']:
+                container = str(containers_list['name'])
+                log.debug('Found container: %s ', container)
+                list_opts_log = ['--timestamps=true', '--tail', str(tail_lines),
+                                 '--namespace', namespace] + since_opt
```

Comment:
I have moved the "since" parameter all the way to this function and generate the 
since_opt declaration in this method.
It was not needed in any of the previous functions...